src/main/java/com/mykaarma/urlshortener/util/UrlServiceUtil.java

```diff
@@ -189,13 +189,12 @@ public void fetchBlackListedWords() throws ShortUrlInternalServerException {
 			
 		String blackListedWordsList = null;
 		try {
-		URL url = new URL(this.blackListedWordsFileUrl);
-		
-		
-		blackListedWordsList = IOUtils.toString(url.openStream(), StandardCharsets.UTF_8);
+			URL url = new URL(this.blackListedWordsFileUrl);
+			blackListedWordsList = IOUtils.toString(url.openStream(), StandardCharsets.UTF_8);
 		}
 		catch(Exception e)
 		{
+			log.error("Error in blackListedWords: ", e);
```

Comment:
Error in fetching blackListedWords file from URL = {} (log the URL along with the exception)